cfme/tests/configure/test_access_control.py

```diff
@@ -128,6 +128,31 @@ def test_user_assign_multiple_groups(appliance, request, group_collection):
         "User's assigned groups are different from expected groups")
 
 
+@pytest.mark.tier(2)
+@pytest.mark.uncollectif(lambda appliance: appliance.version < '5.9')
+def test_user_change_groups(appliance, request, group_collection):
+    """Assign a user to multiple groups and confirm that the user can successfully change groups
+    """
+    group_names = [
+        'EvmGroup-user', 'EvmGroup-user_self_service', 'EvmGroup-desktop']
+
+    group_list = [group_collection.instantiate(description=names) for names in group_names]
```

Comment:
nit-pick on plural vs singular of `names`

Reply:
done